src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,32 @@
-export const TodoFilter = () => (
+type Props = {
+  query: string;
+  onSetSearch: (value: string) => void;
+  onsetQuery: (value: string) => void;
+};
+
+export const TodoFilter:React.FC<Props> = ({
+  query,
+  onSetSearch,
+  onsetQuery,
+}) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
+        <select
+          data-cy="statusSelect"
+          onChange={(event) => onSetSearch(event.target.value)}
+        >
+          <option value="All">
+            All
+          </option>
+
+          <option value="Active">
+            Active
+          </option>
+
+          <option value="Completed">
```

Comment:
Values should be in lower case, won't forget change this values in you state and filter fucntion
```suggestion
          <option value="all">
            All
          </option>

          <option value="active">
            Active
          </option>

          <option value="completed">
```